extensions/tika/runtime/src/main/java/io/quarkus/tika/graalvm/PDFBoxSubstitutions.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.tika.graalvm;
```

Comment:
That looks great, thanks!

Minor nitpick: this class should be in the `io.quarkus.tika.runtime.graal` to be consistent with other extensions.